### PR TITLE
Telling candidates they have an offer with a subject knowledge enhancement (SKE) condition 

### PR DESCRIPTION
**Pull request by MaeveRoseveare**

### CODE REVIEW

File: app/images/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/ske-cover-image.png
Comment (frankieroberto) [inline app/images/apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/ske-cover-image.png:1]: Any chance you could squeeze this image down into more of a landscape format? Means it takes up less height on the page itself, and also avoids it being cropped when shared on Slack, etc.

File: app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  alt: "Illustration with the text 'Offer received' and inforation about doing an 8 week subject knowledge enhancement course in physics"
+
+---
+![Illustration with the text 'Offer received' and inforation about doing an 8 week subject knowledge enhancement course in physics](ske-cover-image.png)
+
+Previously, we designed a way to help training providers add a subject knowledge enhancement (SKE) course as a condition of an offer.
+
+When this offer comes through to the candidate, we need a way for them to see information about the SKE course.
+
+We tested these designs and improved the content shown to candidates. We released this new feature in March 2023.
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md]: May as well add the exact date now?
```suggestion
We tested these designs and improved the content shown to candidates. We released this new feature on 20 March 2023.
```

File: app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+* why they have to do an SKE course
+* how to find an SKE course or who to do it with (SKE providers and ITT providers can be different)
+* SKE courses are free
+* they can get a bursary of £175 a week while they do their SKE course
+* how they can study their course (for example, online, in-person or both)
+
+## What we did
+
+After the training provider sends an offer to a candidate with an SKE condition, we show some guidance before a candidate accepts their offer. This helps explain what an SKE is, so the candidate can make an informed decision about accepting the offer or not.
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md]: Should we always use "SKE course" and not just "SKE"? (Annoying though it is...)

```suggestion
After the training provider sends an offer to a candidate with an SKE course condition, we show some guidance before a candidate accepts their offer. This helps explain what an SKE course is, so the candidate can make an informed decision about accepting the offer or not.
```

File: app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+* the subject and length of their SKE course
+* why they have to do an SKE course
+* when they should complete their SKE course by
+* the SKE course being free and they'll get £175 a week while they do it
+* how they can study their SKE (online, in-person or both)
+
+![Screenshot showing the offer a candidate will recieve with a subject knowledge enhancement course attached. The course is for a 12 week biology course. The content also tells the user they should start their course by April 2023, there is a £175 bursary they can get while doing the course and there are options on how they can study which include full time, part time and online or in-person. The screen then asks the user to respond to the offer'.](ske-offer-page.png)
+
+### language SKE courses
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md]:
```suggestion
### Language SKE courses
```

File: app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+If there are 2 language SKE courses, we show slightly different content with grammatical changes so it makes sense to candidates.
+
+![Screenshot showing the offer a candidate will recieve with 2 language subject knowledge enhancement courses attached. The courses are for an 8 week French course and a 12 week Spanish course. The content also tells the user they should start their courses by April 2023, there is a £175 bursary they can get while doing the courses and there are options on how they can study which include full time, part time and online or in-person. The screen then asks the user to respond to the offer'](language-ske.png)
+
+Once they accept the offer, we show their SKE course with their other conditions, like their Disclosure and Barring Service (DBS) check.
+
+![Screenshot with the heading 'Study before you start training' followed by content saying the user has a 20 weeks maths course with the tag 'Not yet started'. This is followed by content explaining when the user should start their course, that there will be a £175 bursary for the course and to contact the University of Sussex if they have any issues.](offer-page.png)
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md:73]: This bit doesn’t fit within the "Language SKE courses" section. Should it have its own subheading?

File: app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+### Selecting a training provider
+
+Currently, candidates can use the [SKE course directory](https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory) to find an SKE provider.This is published on GOV.UK and not owned by our team.
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md]:
```suggestion
Currently, candidates can use the [SKE course directory](https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory) to find an SKE provider. This is published on GOV.UK and not owned by our team.
```

**Review by frankieroberto — APPROVED** [review]
Nice work! Left some minor suggestions.

File: app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+Overall the designs tested well and there were no major usability issues. Users did make suggestions on where we could make the content clearer.
+
+We iterated the designs to include:
+
+* clarity about how the bursary would be paid – this can vary by provider
+* clarity around what ‘part-time’ means
+* calculating the latest month of when a candidate should start their SKE course so they finish it before their ITT starts
+
+We also tested the SKE course directory and found that most users did not even know it exisited and that they could choose their own SKE provider.
```
Comment (freyakrol1) [inline app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md]:
```suggestion
We also tested the SKE course directory and found that most users did not even know it existed and that they could choose their own SKE provider.
```

File: app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
+---
+title: Telling candidates they need to do a subject knowledge enhancement course
+description: How we designed a way to tell candidates they have a subject knowledge enhancement course as a condition of their offer.
+date: 2023-03-17
+tags:
+  - making an offer
+  - application details
+ogImage:
+  src: /apply-for-teacher-training/telling-candidates-they-have-an-ske-condition/ske-cover-image.png
+  alt: "Illustration with the text 'Offer received' and inforation about doing an 8 week subject knowledge enhancement course in physics"
```
Comment (freyakrol1) [inline app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md]:
```suggestion
  alt: "Illustration with the text 'Offer received' and information about doing an 8 week subject knowledge enhancement course in physics"
```

**Review by freyakrol1 — APPROVED** [review]

**Pull request merged**